### PR TITLE
Mobile workflow

### LINKED ISSUE #886: Implement FxA profile styling, layout
**Drop the [ ... ] menu for now**

Menu items:

* Notes
* Give feedback
* Sign out

Rough mock up

image

### LINKED ISSUE #887: Implement "Last Synced" footer in the navigation drawer
Let's use the same time format we use in the web extension, instead of "24h ago"

image

### LINKED ISSUE #888: Implement Mobile List Style
### [Description]:

For now maybe we can reuse the same timestamp as WebExtension Desktop Notes

image

### LINKED ISSUE #898: Ability to delete Notes from Mobile
We might be able to use one of these for delete "confirmation": https://callstack.github.io/react-native-paper/dialog.html

Delete would go here (top right corner menu): 
image

### LINKED ISSUE #921: Ability to update notes from mobile
Changes made to the note in the webview should be pushed to Kinto server

Ref: https://github.com/mozilla/notes/issues/898

### LINKED ISSUE #950: Clear Kinto on Logout in mobile
Clear Kinto on Logout in mobile

### LINKED ISSUE #953: Instead of calling fxaUtils.fxaGetCredential() pull that info from react / redux state
From https://github.com/mozilla/notes/pull/940

### LINKED ISSUE #954: Mobile Editor: When creating a fresh note, wrap the new text into a <p></p>
Currently `contenteditable` is not inserting a paragraph. Paragraphs only start after a new line.

This results in the WebExtension not being able to populate the list view

image

### PR DESCRIPTION
**Pull request by sebastienbarbier**
- [x] Redesign drawer and listPanel
- [x] Create, update, and delete notes
- [x] Fix performance issue
- [x] Sync with kinto

Fix #954, fix #921, fix #898, fix #888, fix #887, fix #886, fix #950, and fix #953.

### CODE REVIEW

**Comment by vladikoff** [comment]
I filed other follow up bugs, but let's not worry about those for now...

**Comment by vladikoff** [comment]
I pushed this change https://github.com/vladikoff/react-native-zss-rich-text-editor/commit/78b33809f5f53f4e17e60d41df32d015718b58f2#diff-5764eca5893a91bd5fb61afbae89595dR879 to use <p> instead of <divs> . However that changes doesn't wrap the first line of the Note, we will either need to fix this in the editor or insert a blank paragraph when we start a fresh note

File: native/app/components/ListItem.js [hunk 1]
```diff
@@ -2,53 +2,116 @@ import React from 'react';
...
-      }>
-        <View>
-          <View style={{ padding: 10 }} >
-            <Text>
-              {striptags(content).substr(0, 50)}
-            </Text>
-            <Text>
-              {formatLastModified(lastModified)}
-            </Text>
+      <TouchableRipple onPress={this._navigateToNote} >
```
Comment (vladikoff) [inline native/app/components/ListItem.js]: I don't see the Ripple actually happen? Do we need extra styles or use TouchableOpacity ?

Comment (sebastienbarbier) [reply native/app/components/ListItem.js]: I need to investigate further why ripple do not happen when onPress have a navigate event. If it's not easy to fix we'll move back to TouchableOpacity.

File: native/app/utils/sync.js [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
 
 const browser = {
+  i18n: {
+    getMessage: () => {
+      return '';
+    }
+  },
   runtime: {
     sendMessage: () => {
       console.log('sendMessage', arguments);
```
Comment (vladikoff) [inline native/app/utils/sync.js]: This is adding a lot of logging now into the console...

**Comment by vladikoff** [comment]
- [ ] Sync after note is updated?
- [ ] Better error handling?

File: native/app/utils/sync.js (hunk 2 above)
Comment (vladikoff) [reply native/app/utils/sync.js]: Might have to remove this line

File: native/app/components/ListItem.js (hunk 1 above)
Comment (sebastienbarbier) [reply native/app/components/ListItem.js]: Going back with TouchableOpacity 👍

File: native/app/browser.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    browser.runtime.onMessage.dispatch(args[1]);
+  } else {
+    browser.runtime.onMessage.dispatch(args[0]);
+  }
+}
+
+const browser = {
+  i18n: {
+    getMessage: () => {
+      return '';
```
Comment (vladikoff) [inline native/app/browser.js]: Ref: follow up issue https://github.com/mozilla/notes/issues/957

**Pull request merged**